modules/appeals_api/spec/support/rswag_config.rb

```diff
@@ -436,6 +436,8 @@ def hlr_create_schemas
 
   def decision_reviews_hlr_response_schemas
     schemas = hlr_response_schemas
+    schemas = deep_replace_key(schemas, :createDate, :createdAt)
```

Comment:
I think I may have caught something after I approved this PR. For NODs and SCs, you're updating the `nod_response_schemas` and `sc_response_schemas` respectively, but for HLRs, you're updating `decision_reviews_hlr_response_schemas`, which makes it seem like the field name change would affect V2 but not V0. Is the following correct instead, or am I missing something?

```
def hlr_response_schemas
  schemas = decision_reviews_hlr_response_schemas
  schemas = deep_replace_key(schemas, :createDate, :createdAt)
```

Reply:
Oh this one actually goes in reverse of the others:
- In decision reviews, there is a contestable issues endpoint in the HLR namespace that we haven't brought into the HLR v0 API (since we have the separate appealable issues v0 API that offers the same functionality)
- The only other schema in HLR besides the one for that contestable issues endpoint (whether decision reviews or v0) is the `showHlr` schema, which is actually used for presenting the HLR at the `create` endpoint
- In the case of HLR and no others, the decision reviews rswag config for `showHlr` from the v0 API is reused for decision reviews, then it adds on the contestable issues endpoint schema. This is why we replace `createDate` with `createdAt` in this case - the schema comes from the newer API and is reused in decision reviews where the `createDate` changes haven't been applied. You can see how this works out in the rendered swagger.json files for decision reviews (no changes to their existing `createdAt`, `updatedAt` names there)

Reply:
Ah, okay! That makes sense. Thanks for the explanation! I'm going to approve the PR again.